packages/workspace/src/utils/runtime-lint-utils.spec.ts

```diff
@@ -0,0 +1,190 @@
+import { checkCircularPath } from './runtime-lint-utils';
+
+function transformGraph(graph) {
+  const nodes = graph.nodes.reduce((acc, name) => {
+    return {
+      ...acc,
+      [name]: { name }
+    };
+  }, {});
+
+  const dependencies = Object.keys(graph.dependencies).reduce((acc, dep) => {
+    const targets = graph.dependencies[dep].reduce((acc, name) => {
+      return [...acc, { source: dep, target: name }];
+    }, []);
+
+    return {
+      ...acc,
+      [dep]: targets
+    };
+  }, {});
+
+  return {
+    nodes,
+    dependencies
+  };
+}
+
+function getPath(graph, node) {
+  return checkCircularPath(graph as any, { name: node.to } as any, { name: node.from } as any);
+}
+
+describe('should find the path between nodes', () => {
+  it('should return empty path when when there are no connecting edges', () => {
+    /*
+
+    A -> B -> C - > E
+
+     */
+
+    const graph = {
+      nodes: ['A', 'B', 'C', 'E'],
```

Comment:
I think you can probably make it a graph with 2 nodes and 1 edge. Based on the assertion, C & E play no role.